openedx/core/djangoapps/verified_track_content/tests/test_partition_scheme.py

```diff
@@ -53,10 +53,9 @@ def test_multiple_groups(self):
         self.assertIsNotNone(self.get_group_by_name(partition, "Verified Enrollment Track"))
         self.assertIsNotNone(self.get_group_by_name(partition, "Credit Mode"))
 
-    def test_to_json_not_supported(self):
+    def test_to_json_supported(self):
         user_partition = create_enrollment_track_partition(self.course)
-        with self.assertRaises(TypeError):
-            user_partition.to_json()
+        self.assertIsNotNone(user_partition.to_json())
```

Comment:
Instead of just asserting that it isn't None, can you test something about what it should be?

I would call it test_to_json, and assert what you expect the output to be.